### PR TITLE
Resolver support both submitters

### PR DESCRIPTION
**Pull request by typhoonzero**
implementing https://github.com/sql-machine-learning/sqlflow/pull/507 for `codegen.go`.

TODO: need to update text classification document to use this column grammar.

### CODE REVIEW

**Comment by tonyyang-svail** [comment]
@typhoonzero need to resolve conflicts since https://github.com/sql-machine-learning/sqlflow/pull/528 has been merged. :)

**Review by wangkuiyi — APPROVED** [review]

**Comment by wangkuiyi** [comment]
It seems that this PR is 9 days old. And it seems active, resolving comments, and passed CI, so I am giving approval without looking deeply into the code. Please others also comment.

File: scripts/test.sh [hunk 1]
```diff
@@ -18,6 +18,10 @@ set -e
 service mysql start
 
 export SQLFLOW_TEST_DB=mysql
+# NOTE: we have already installed sqlflow_submitter under python installation path
+# using latest develop branch, but when testing on CI, we need to use the code in
+# the current pull request.
+export PYTHONPATH=$GOPATH/src/github.com/sql-machine-learning/sqlflow/sql/python
```
Comment (tonyyang-svail) [inline scripts/test.sh:24]: Good catch!

**Review by tonyyang-svail — APPROVED** [review]
LGTM!

**Pull request merged**